test/jdk/javax/swing/JColorChooser/Test4222508.java

```diff
@@ -21,36 +21,47 @@
  * questions.
  */
 
+import java.awt.BorderLayout;
+import javax.swing.JCheckBox;
+import javax.swing.JColorChooser;
+import javax.swing.JFrame;
+
 /*
  * @test
  * @bug 4222508
+ * @library /test/jdk/java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
  * @summary Tests the color chooser disabling
- * @author Sergey Malenkov
- * @run applet/manual=yesno Test4222508.html
+ * @run main/manual Test4222508
  */
+public final class Test4222508 {
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test4222508")
+                .instructions("Click on colors in the JColorChooser. Then uncheck the checkbox \n" +
+                        "and click on colors again. \n" +
+                        "If the JColorChooser is disabled when the checkbox is unchecked, \n" +
+                        "then pass the test.")
+                .rows(5)
+                .columns(40)
+                .testTimeOut(10)
+                .testUI(Test4222508::test)
+                .build()
+                .awaitAndCheck();
+    }
 
-import java.awt.BorderLayout;
-import java.awt.event.ItemEvent;
-import java.awt.event.ItemListener;
-import javax.swing.JApplet;
-import javax.swing.JCheckBox;
-import javax.swing.JColorChooser;
-
-public final class Test4222508 extends JApplet implements ItemListener {
+    public static JFrame test() {
+        JFrame frame = new JFrame();
```

Comment:
give the frame a meaningful title that is not identical to the one to the PassFailJFrame.